lib/worker.js

```diff
@@ -30,7 +38,7 @@ async function start () {
     // The error codes may change based on the node.js version (ENOTDIR > 12, ERR_MODULE_NOT_FOUND <= 12 )
     if ((error.code === 'ENOTDIR' || error.code === 'ERR_MODULE_NOT_FOUND') &&
      workerData.filename.startsWith('file://')) {
-      fn = realRequire(workerData.filename.replace('file://', ''))
+      fn = realRequire(workerData.filename.replace(workerData.filename[9] === ":" ? 'file:///' : 'file://', ''))
```

Comment:
This seems very fragile. I think the new code is looking for `file://c:/foo/bar`. But there is no guarantee the provided path will be absolute. I think a proper `process.platform` check should be used instead.

Reply:
I agree this is fragile. I agree to use process.platform

The old code removes the file:// from the path. in unix systems. file:///example/ex.js will be : /example/ex.js. But in windows file:///C:/example/ex.js will be : /C:/example.js and this will make the module loading fail to resolve the file.